ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -178,7 +178,7 @@ Verify RHODS Installation
     ...    label_selector=app.kubernetes.io/part-of=trainingoperator   timeout=400s
   END
 
-  IF    "${dashboard}" == "true" or "${workbenches}" == "true" or "${modelmeshserving}" == "true" or "${datasciencepipelines}" == "true" or "${kserve}" == "true" or "${kueue}" == "true" or "${codeflare}" == "true" or "${ray}" == "true" or "${trustyai}" == "true" or "${modelregistry}" == "true" or "${trainingoperator}" == "true"    # robocop: disable
+  IF    "${dashboard}" == "true" or "${workbenches}" == "true" or "${modelmeshserving}" == "true" or "${datasciencepipelines}" == "true" or "${kserve}" == "true"  or "${kueue}" == "true"  or "${codeflare}" == "true" or "${ray}" == "true" or "${trustyai}" == "true" or "${modelregistry}" == "true" or "${trainingoperator}" == "true"    # robocop: disable
```

Comment:
https://github.com/red-hat-data-services/ods-ci/pull/2016#discussion_r1836934793

Reply:
damn, I did a rebase and lost all the changes. Will fix it next week